src/main/java/io/cdap/plugin/salesforce/plugin/sink/batch/StructuredRecordToCSVRecordTransformer.java

```diff
@@ -32,61 +29,16 @@
 
   public CSVRecord transform(StructuredRecord record) {
     List<String> fieldNames = new ArrayList<>();
-    List<Object> values = new ArrayList<>();
+    List<String> values = new ArrayList<>();
 
     for (Schema.Field field : record.getSchema().getFields()) {
       String fieldName = field.getName();
-      Object value = convert(record.get(fieldName), field);
+      String value = SalesforceTransformUtil.convertSchemaFieldToString(record.get(fieldName), field);
 
       fieldNames.add(fieldName);
       values.add(value);
     }
 
-    CSVRecord csvRecord = new CSVRecord(fieldNames, values);
-    return csvRecord;
-  }
-
-  private static Object convert(Object value, Schema.Field field) {
```

Comment:
I think this method belong in this class more than in some util class. Let's make it public (as well as changing the return type to String).

Reply:
done